mscore/editstyle.cpp

```diff
@@ -43,15 +43,6 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
       setObjectName("EditStyle");
       setupUi(this);
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
-
```

Comment:
I moved this code to the bottom of the constructor, right before restoreGeometry. That's a better place for it, because it directly affects the code's ability to restore a small window size. That is the only existing part of the constructor that is directly related to this sizing code.